doc/installation.md

```diff
@@ -28,7 +28,7 @@ $ chmod a+x kerl
 $ mv kerl /usr/bin
 $ kerl list releases
 $ kerl build 18.2 r18.2 # this takes a while
-$ kerl install r18.2 /usr/lib/erlang
+$ kerl install r18.2 /usr/local/erlang
```

Comment:
I'll offer a dissenting opinion - I think OS packages and kerl should be mutually exclusive. I never install OS packages so putting kerl-based builds in the "normal" location makes sense to me. The installation instructions are for setting up a development machine with which to hack on Kazoo. I would again encourage using kerl to install latest and greatest Erlang version.

Remember, this guide is for getting up and running somewhat quickly, not for covering "advanced" scenarios where you need kerl + OS packaged Erlang or multiple versions of kerl-installed Erlang versions.